vancouver-imperial-college-london.csl

```diff
@@ -256,8 +254,16 @@
   <macro name="journal-location">
     <choose>
       <if type="article-journal article-magazine review review-book" match="any">
-        <text variable="volume"/>
-        <text variable="issue" prefix="(" suffix=")"/>
+        <choose>
```

Comment:
this seems unnecessarily complex -- what is it designed to fix/address?

Reply:
This is designed to distinguish the issue number , where it exists, from the volume by placing it in parenthesis

Reply:
But that's not what it's doing: the issue is already in parentheses with the previous code. For some reason you're checking if volume and issue are numbers? That seems particularly problematic for issue, which you also see as "Fall", e.g.

Reply:
Hi, this has been simplified.